default.json5

```diff
@@ -119,6 +95,7 @@
       matchUpdateTypes: [
         "minor",
       ],
+      matchCurrentVersion: "!/^0/",
```

Comment:
So what happens to minor 0.x versions? I think they are just ignored now.
I propose this instead of having multiple `matchCurrentVersions`:

```
{
  matchCurrentVersion: "!/^0/",
  dependencyDashboardApproval: true
}
```
This is added at the end of the config.

Reply:
added the change, but the filter in this case needs to be `"/^0/"`